doc/README.md

```diff
@@ -0,0 +1,8 @@
+# InfiniTime Documentation
+
+The /doc folder is organized as follows:
+
+ * [dev/](./dev): Information for developers looking to contribute
+ * [ref/](./ref): Technical references for InfiniTime and the PineTime
+ * [user/](./user): User-facing documentation (How to use InfiniTime)
```

Comment:
We don't have a markdown formatter or explicit rules yet, but if we did, bullet points would be written like so.

```suggestion
- [dev/](./dev): Information for developers looking to contribute
- [ref/](./ref): Technical references for InfiniTime and the PineTime
- [user/](./user): User-facing documentation (How to use InfiniTime)
```

Reply:
Updated